tests/test_pipelines_automatic_speech_recognition.py

```diff
@@ -368,6 +373,25 @@ def test_with_lm_fast(self):
         self.assertEqual(output, [{"text": ANY(str)}])
         self.assertEqual(output[0]["text"][:6], "<s> <s")
 
+    @require_torch
+    @require_pyctcdecode
+    def test_with_local_lm_fast(self):
+        speech_recognizer = pipeline(
+            model=snapshot_download("hf-internal-testing/processor_with_lm"),
+        )
```

Comment:
You need to to:

```suggestion
        local_dir = snapshot_download("hf-internal-testing/processor_with_lm")
        speech_recognizer = pipeline(
            task="automatic-speech-recognition", model=local_dir,
        )
```

the task cannot be inferred from local directories. That information is created for you when you create a model on the hub, the local directories doesn't have "tags" like on the hub.